tests/stix_shifter/test_qradar_json_to_stix.py

```diff
@@ -123,7 +124,7 @@ def test_cybox_observables(self):
         assert(curr_obj.keys() == {'type', 'value'})
         assert(curr_obj['value'] == 'example.com')
 
```

Comment:
We should have a test showing that two network-traffic objects were created, and that the source and destination references point to the correct objects (either ipv4 or mac address).